src/LoadableModules.cc

```diff
@@ -7,8 +7,8 @@
  */
 
 #include "squid.h"
-#include "Debug.h"
-#include "DebugMessages.h"
+#include "debug/Stream.h"
+#include "debug/Messages.h"
```

Comment:
Please run ./scripts/source-maintenance.sh on this branch. There are more than a few of these includes which should change their ordering with the new names.

Reply:
Commit f580a4c.

As you probably know, ./scripts/source-maintenance.sh is not stable enough (yet) for developers to run against their PR branches -- it produces too many unwanted changes, forcing the developer to waste time undoing them. We should not ask to run it.

Reply:
That period of many unrelated changes should be over now as of this Jan enforcement cycle initiating the HERE purge.

IMO it is reasonable for a reviewer to ask any dev to run it if the PR submission contains a large amount of tiny style issues that it can erase without needing to review-comment specifics. Keeping the official code compliant is one of the things that falls on maintainer shoulders if these things get past review.

Reply:
> That period of many unrelated changes should be over now as of this Jan enforcement cycle initiating the HERE purge.

I was not talking about changes to ./scripts/source-maintenance.sh. I was talking about the lack of stability of that script _results_. They are unstable -- two executions of the script may (and do) edit two identical source files differently, resulting in unwanted formatting changes.

There are also problems with gperf and seasonal problems with copyright years. We are slowly fixing the script problems, but it is not yet ready for production use.

There is also a scope problem: Until we start rejecting malformed PRs, it is wrong to ask one developer to format code already deformed by recently merged PRs. The test enforcing PR formatting may be ready (IIRC, it has been running on Sempahore CI for more than a year, but it overwrites the failure result with a success). However, we cannot fully enable this enforcement until the quality of ./scripts/source-maintenance.sh is improved.

> IMO it is reasonable for a reviewer to ask any dev to run it

Not until the script is reliable _and_ we start enforcing formatting before each PR is merged IMO.